docker/Dockerfile.prod

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.15
```

Comment:
For production, let's use `multi-stage` builds. We don't need most of the stuff once we get compiled version of the app and assets.

I would recommend https://github.com/GoogleContainerTools/distroless images and especially `gcr.io/distroless/static` because go is static so let's make the image as small as possible. This image is better than `scratch` because it comes with many security aspects built in